src/lib.rs

```diff
@@ -56,31 +54,32 @@ mod env;
 mod error;
 mod log_util;
 mod metadata;
+mod not_sync;
```

Comment:
What about `task` or `executor`? This module impls a custom future executor based on grpc.

Reply:
I changed it to `task` since it's shorter than `executor`. Good suggestion.